src/test/kotlin/com/icat/orboarding/user/adapters/outbound/persistence/mysql/UserPersistenceTest.kt

```diff
@@ -61,4 +61,18 @@ internal class UserPersistenceTest {
 
         assertTrue(user.isEmpty)
     }
+
+    @Test
+    fun `should update a user by current email`() {
+        val currentEmail = "kaike@gmail.com"
+        val newEmail = "kaike@gmail.com"
+        val currentUser = userPersistence.getUser(currentEmail).get()
+        val updatedUser = userPersistence.updateUser(UserDomain(id = currentUser.id, email = newEmail, password = "321"))
+
+        assertEquals(currentUser.id, updatedUser.id)
+        assertEquals(currentEmail, currentUser.email)
+        assertEquals(newEmail, updatedUser.email)
+        assertEquals("321", updatedUser.password)
+        assertNotEquals(currentUser.updatedAt, updatedUser.updatedAt)
```

Comment:
Manin, n sei se vai ser útil pq só fiz no Java, mas só por curiosidade: Tinha visto uma forma de comparar se todos os campos de dois objetos são iguais usando algo assim:
`assertThat(actualObject).usingRecursiveComparison().isEqualTo(expectedObject)`, pra evitar fazer asserts em cada campo individualmente.

Tem como configurar pra ignorar alguns campos específicos também, exemplo aqui: https://stackoverflow.com/questions/43316133/how-to-compare-recursively-ignoring-given-fields-using-assertj

Reply:
Topzera demais usar esse assertThat maninho!
Vou incluir a biblioteca pra gente usar no projeto. Não sabia que dava para ignorar alguns campos! Top demais, obrigado por compartilhar!!!!

Reply:
Ou melhor maninho, vou incluir em outro PR, e utilizar ele nos testes unitários.
Nesse teste aqui, não tem um objeto pronto que esta esperando receber as informações. Já no teste unitário, como temos mockado um resultado esprado, podemos utilizar essa lib como ferramenta.